src/screens/organization/repository-list.js

```diff
@@ -0,0 +1,97 @@
+/* eslint-disable no-shadow */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import { RepositoryList } from 'components';
+
+import { getRepos } from 'api/rest/providers/github/endpoints/orgs';
+import { searchRepos } from 'api/rest/providers/github/endpoints/search';
```

Comment:
How about doing something like this?

```js
import { getOrganizationRepos, searchRepos } from 'api/rest/providers/github/endpoints';
```

Reply:
I would like to have an `orgs` prefix for organization related methods (like a namespace, but without the '.')

I inspired my naming from https://octokit.github.io/node-github/ which seems to me like it's nailing it.

Reply:
This comment should be considered with [this](https://github.com/gitpoint/git-point/pull/457#discussion_r144004561).

If we assume that there will be other providers, including.